deployment/src/main/java/com/tietoevry/quarkus/resteasy/problem/deployment/ProblemProcessor.java

```diff
@@ -25,62 +25,59 @@
     private static final String FEATURE_NAME = "resteasy-problem";
     private static final String PACKAGE = "com.tietoevry.quarkus.resteasy.problem.";
```

Comment:
Maybe `INTERNAL_PACKAGE`

Reply:
Agree, `PACKAGE` is not very descriptive. 
Another option: `EXTENSION_MAIN_PACKAGE` ?